app/src/main/res/values/syles.xml

```diff
@@ -12,4 +12,17 @@
         <item name="android:textSize">@dimen/size_main_activity_button</item>
         <item name="android:backgroundTint">@color/pomegranate_button_main_activity</item>
     </style>
+
+    <style name="TitleConfigureParkingDialogStyle">
+        <item name="android:layout_centerHorizontal">true</item>
```

Comment:
it not make sense to use a style with only one property, please remove this style

Reply:
Done!